### PR TITLE
Adjust consensus and reward of DPOS node after ChangeCommitteeNewCRHeight

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

**Review by xiaomingloveqinger — APPROVED** [review]

File: blockchain/confirmvalidator.go [hunk 1]
```diff
@@ -179,6 +209,26 @@ func ProposalSanityCheck(proposal *payload.DPOSProposal) error {
...
+func IllegalProposalContextCheck(proposal *payload.DPOSProposal) error {
+	arbiters := DefaultLedger.Arbitrators.GetAllProducersPublicKey()
+	var isArbiter bool
+	for _, a := range arbiters {
+		pk, err := hex.DecodeString(a)
+		if err != nil {
+			continue
+		}
+		if bytes.Equal(pk, proposal.Sponsor) {
+			isArbiter = true
```
Comment (Houshoupei84) [inline blockchain/confirmvalidator.go:221]: Maybe we can break from here

File: blockchain/confirmvalidator.go [hunk 2]
```diff
@@ -256,6 +306,25 @@ func VoteSanityCheck(vote *payload.DPOSProposalVote) error {
...
+func IllegalVoteContextCheck(vote *payload.DPOSProposalVote) error {
+	arbiters := DefaultLedger.Arbitrators.GetAllProducersPublicKey()
+	var isArbiter bool
+	for _, a := range arbiters {
+		pk, err := hex.DecodeString(a)
+		if err != nil {
+			continue
+		}
+		if bytes.Equal(pk, vote.Signer) {
+			isArbiter = true
```
Comment (Houshoupei84) [inline blockchain/confirmvalidator.go:318]: Here is the same as above

File: blockchain/confirmvalidator.go (hunk 1 above)
Comment (RainFallsSilent) [reply blockchain/confirmvalidator.go:221]: Fixed

File: blockchain/confirmvalidator.go (hunk 2 above)
Comment (RainFallsSilent) [reply blockchain/confirmvalidator.go:320]: Fixed

**Review by Houshoupei84 — APPROVED** [review]

**Review by peterwillcn — APPROVED** [review]

**Pull request merged**